src/reducers/filterReducer.js

```diff
@@ -69,23 +78,14 @@ export const ticketId = (defaultTicketId) => (state = defaultTicketId, action) =
   }
 }
 
-const initialFilter = {
-  ticketId: null,
-  incidentId: null,
-  eventTypes: null,
-  startTime: null,
-  endTime: null,
-  DataSearch: null
-}
-
 export const filter = (defaultFilter = initialFilter) => {
```

Comment:
Since initialFilter value is assigned to child reducers, it need not to be assigned to defaultFilter here.

Reply:
I see what you mean. I'll still leave it there because it covers the case where defaultFilter is undefined.